resources/developer-mode.mjs

```diff
@@ -15,11 +15,16 @@ export function createDeveloperModeContainer() {
     content.append(createUIForSuites());
 
     content.append(document.createElement("hr"));
+    content.append(createUIForIterationCount());
+    content.append(document.createElement("br"));
```

Comment:
(optional for this patch)
Instead of these br elements, can't we just use some CSS margin?

Actually I looked with the devtools, they don't seem useful at all. Am I missing something?

Reply:
I also look at using `display: flex` or `inline-flex` to avoid the `float: right` you're adding below (`float` isn't great for layout with all its quirks, now that we can use `flex`), but there seems to be more changes needed for that, so let's defer this for a future PR (I might have a look myself).

Reply:
done, switched over to flex.